### PR TITLE
Soback 132 gephi export

### PR DESCRIPTION
**Pull request by jesperlauridsen**
Added searchbox for gephi export.
Moved function to fix height to mixin.
Added proxy path to vue.config.js.
Added text for query examples, limitations and quick gephi guide.
styled table and setup a little bit.

@jorntx for code review
@thomasegense for functionality

### CODE REVIEW

**Review by thomasegense — CHANGES_REQUESTED** [review]
Great layout!
A few text changes. (some my own mistakes...)
2) overview tab  >- 2) Overview tab
giant compontent -> giant component 

Add a point 9:
9) Use Graph Presenter(https://github.com/statsbiblioteket/graph_presenter) to convert the SVG into a zoomable image.
(Do not show the link, make Graph Presenter as link text.

Under limitations have this line with a larger font:
Examples of linkgraphs create with SolrWayback, Gephi and Graph Presenter can be seen here 
Linkgraphs examples from the Danish Netarchive (https://labs.statsbiblioteket.dk/linkgraph/)
Again as a link with the text.

File: src/js/src/mixins/SearchboxUtils.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export default {
+  methods: {
+    $_getSizeOfTextArea(id) {
+      let textarea = document.getElementById(id)
+      textarea.style.height = '1px'
+      textarea.style.height = (textarea.scrollHeight) + 'px'
```
Comment (jorntx) [inline src/js/src/mixins/SearchboxUtils.js:6]: Is the parenthesis needed here?

**Review by jorntx — APPROVED** [review]
All is good! Have one minor question.

The instructions do not make sense to me in relation to the export but I guess that this is due to my very small dataset.

**Review by jorntx — APPROVED** [review]
Approved! Thanks for keeping an eye on the ngram search box 👍

**Review by thomasegense — APPROVED** [review]

**Pull request merged**